src/components/DatePicker/DatePicker.tsx

```diff
@@ -14,9 +14,11 @@ import classNames from 'classnames'
 /**
  * @see https://github.com/Hacker0x01/react-datepicker/issues/1333#issuecomment-2363284612
  */
+/* eslint-disable @typescript-eslint/no-explicit-any */
 const ReactDatePicker = (((RDP.default as any).default as any) ||
   (RDP.default as any) ||
   (RDP as any)) as typeof RDP.default
+/* eslint-enable @typescript-eslint/no-explicit-any */
```

Comment:
This is one of the few places where it is not obvious how to remove `any`.

I think we need to eventually move away from / upgrade `ReactDatePicker`. The version of the package we're on has strange types because the whole thing was written in JS and augmented by an @types package.